idea/testData/quickfix/nullables/unsafeInfixCall/unsafeInvokeWithImplicitReceiver.kt

```diff
@@ -0,0 +1,15 @@
+// "Replace with safe (?.) call" "false"
+// ACTION: Add non-null asserted (!!) call
+// ACTION: Convert to expression body
+// ACTION: Replace with safe (this?.) call
+// ACTION: Wrap with '?.let { ... }' call
+// ERROR: Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type A?
+
+class A {
+    fun foo() {
+    }
+}
+
+fun A?.bar() {
+    <caret>foo()
+}
```

Comment:
Why does this test have so strange name? I see no `invoke` here, either safe or unsafe